### PR TITLE
DECISION-17125-breadcrumb-ui-change

### PR DESCRIPTION
**Pull request by srisrim**
Height fixed for the RFV list drop down in breadcrumb

### CODE REVIEW

**Comment by maayanmessica** [comment]
@srisrim The fix looks good but I see in the Jira ticket that they wanted the fix to be a scrollbar from 5 items and more. Not 4.

**Comment by srisrim** [comment]
> @srisrim The fix looks good but I see in the Jira ticket that they wanted the fix to be a scrollbar from 5 items and more. Not 4.

Correct @maayanmessica this fix for the scroll to appear when dropdown has more than 4 items and not when it has only 4 items.

**Comment by maayanmessica** [comment]
image
This is what I got when testing it.

**Comment by srisrim** [comment]
> image
> This is what I got when testing it. You can ignore the error message.

Yes because the fix i gave is not reflecting here. We tried together last week remember ? you asked me to give the fix and you will be checking for the fix to work.

**Comment by maayanmessica** [comment]
@srisrim I wasn't sure if you checked it when changing it in the UI when you showed me. Anyway, no problem you can fix it and I will approve.

File: libs/angular-breadcrumb/src/breadcrumb/component/breadcrumb.component.less [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @popup-row-height: 40;
-@popup-number-of-rows: 4;
+@popup-number-of-rows: 5;
 @popup-height: @popup-row-height * @popup-number-of-rows;
```
Comment (maayanmessica) [inline libs/angular-breadcrumb/src/breadcrumb/component/breadcrumb.component.less]: @srisrim Can you revert this commit. Looks like you were right, I checked again now with the change and it worked with 4. I don't know what happened before.

**Pull request merged**